### PR TITLE
Add User model

### PR DESCRIPTION
**Pull request by alphawolf07**

### CODE REVIEW

**Comment by Aritra8438** [comment]
Great PR!
Things I want to mention:
- There are changes in `requirements.txt`. Run `pip freeze > requirements.txt` in the root directory and commit it as well. 
- For now, let's keep it to sqlite3. We will host `postgres database` at **Neon** later.

It seems like, you are working on forked repo. No problem with that, but you have the access to the main repo as well.

File: backend/settings.py [hunk 1]
```diff
@@ -78,8 +82,12 @@
...
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE':'django.db.backends.postgresql_psycopg2',
+        'NAME':'Sample3',
+        'USER':'postgres',
+        'PASSWORD':'root',
+        'HOST':'localhost',
+        'PORT':'5432',
     }
```
Comment (Aritra8438) [inline backend/settings.py:91]: Let's keep sqlite3 for now.

**Comment by alphawolf07** [comment]
Okay I'll do the requirements.txt part. As for the postgres, I'm doing it
in my branch for my convenience but I'll change it to sqlite before
committing.

**Review by Aritra8438 — CHANGES_REQUESTED** [review]
Commit `requirements.txt`.

**Review by Aritra8438 — APPROVED** [review]

**Pull request merged**